app/assets/stylesheets/custom.css.scss

```diff
@@ -581,3 +581,145 @@ header.navigation {
   }
 }
 
+article.type-system-slab {
+  ///////////////////////////////////////////////////////////////////////////////////
```

Comment:
Rule sets should be ordered as follows: `@extends`, `@includes` without `@content`, properties, `@includes` with `@content`, nested rule sets